security/pkg/k8s/chiron/controller.go

```diff
@@ -297,15 +298,8 @@ func (wc *WebhookController) scrtUpdated(oldObj, newObj interface{}) {
 		return
 	}
 
-	certLifeTimeLeft := time.Until(cert.NotAfter)
-	certLifeTime := cert.NotAfter.Sub(cert.NotBefore)
-	// Because time.Duration only takes int type, multiply gracePeriodRatio by 1000 and then divide it.
-	gracePeriod := time.Duration(wc.gracePeriodRatio*1000) * certLifeTime / 1000
-	if gracePeriod < wc.minGracePeriod {
-		log.Warnf("gracePeriod (%v * %f) = %v is less than minGracePeriod %v. Apply minGracePeriod.",
-			certLifeTime, wc.gracePeriodRatio, gracePeriod, wc.minGracePeriod)
-		gracePeriod = wc.minGracePeriod
-	}
+	certUtil := certutil.NewCertUtil(int(wc.gracePeriodRatio * 100))
```

Comment:
Perhaps you can move certUtil into wc *WebhookController as a new member, so that we don't need to call NewCertUtil, and create/destroy certUtil frequently.

Reply:
Done. Please take another look.